cmd/ocm-backplane/managedJob/logsManagedJob.go

```diff
@@ -56,10 +62,22 @@ func newLogsManagedJobCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
+
 			bpCluster, err := utils.DefaultClusterUtils.GetBackplaneCluster(clusterKey)
 			if err != nil {
 				return err
 			}
+
+			if managerFlag {
+				if mcid, clusterName, _, err := ocm.DefaultOCMInterface.GetManagingCluster(bpCluster.ClusterID); err == nil {
+					bpCluster, err = utils.DefaultClusterUtils.GetBackplaneCluster(mcid)
+					_ = clusterName
```

Comment:
Instead of doing this here, you could change line 72 above so that `clusterName` is replaced with `_`.